### PR TITLE
Automated code clean up

### PR DESCRIPTION
**Pull request by Loobinex**
Moves variables to the scope where they actually belong.

### CODE REVIEW

**Comment by Loobinex** [comment]
@spartahawk and @Trass3r do you agree with Trass3r that the code is 'cleaner' like this, or do you prefer the original method?

**Comment by spartahawk** [comment]
Yes, more glance-able and follows clean code principles better (adding curly braces for all the one-liner next line if statements would be good too).

There's no way I'm going to look through all of it to make sure it never makes a change that might break something though. Be sure everything works as expected. Have you checked that game saves work, or whatever it was?

**Comment by Loobinex** [comment]
I get that. And no, I've yet to test it. But that's the thing 'everything working as expected' is a bit much too. There's no automated testing on this thing.

**Comment by spartahawk** [comment]
Having said that about cleaner code, I've never been in such a hurry with this code base that I needed those extra seconds to look a few lines back to check a variable. If the automated tool that did this were already well-versed, I'd say, sure, do it. But at the risk of changes we can't be sure of, combined with a lack of current code being any real problem, I don't think this is overall better.

**Comment by spartahawk** [comment]
*well-vetted

File: src/config_trapdoor.c [hunk 1]
```diff
@@ -853,10 +848,9 @@ TbBool load_trapdoor_config(const char *conf_fname, unsigned short flags)
 {
     static const char config_global_textname[] = "global traps and doors config";
     static const char config_campgn_textname[] = "campaign traps and doors config";
-    char *fname;
-    TbBool result;
-    fname = prepare_file_path(FGrp_FxData,conf_fname);
-    result = load_trapdoor_config_file(config_global_textname,fname,flags);
+
```
Comment (Loobinex) [inline src/config_trapdoor.c]: @Trass3r are all these empty lines intended? I highlight just one, but happens often.

File: src/config_trapdoor.c [hunk 2]
```diff
@@ -490,14 +489,12 @@ TbBool parse_trapdoor_trap_blocks(char *buf, long len, const char *config_textna
 
 TbBool parse_trapdoor_door_blocks(char *buf, long len, const char *config_textname, unsigned short flags)
 {
-  struct ManfctrConfig *mconf;
   struct DoorConfigStats *doorst;
-  long pos;
-  int i,k,n;
-  int cmd_num;
+
```
Comment (Loobinex) [inline src/config_trapdoor.c]: @Trass3r second example.
I played a few games with this pull, and did not run into problems.

File: src/config_trapdoor.c (hunk 1 above)
Comment (Trass3r) [reply src/config_trapdoor.c]: Not really. It does run clang-format on the patches.

Comment (Loobinex) [reply src/config_trapdoor.c]: Sorry, I don't know what that means.

Comment (Trass3r) [reply src/config_trapdoor.c]: It's a tool to automatically format code.

**Pull request merged**